cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -266,8 +275,8 @@ func generateGenesis(ctx context.Context) (state.BeaconState, error) {
 		//gen.Config.CancunTime = interop.GethCancunTime(f.GenesisTime, params.BeaconConfig())
 		gen.Config.CancunTime = interop.GethCancunTime(f.GenesisTime, params.BeaconConfig())
 		log.
-			WithField("shanghai", gen.Config.ShanghaiTime).
-			WithField("cancun", gen.Config.CancunTime).
+			WithField("shanghai", fmt.Sprintf("%d", *gen.Config.ShanghaiTime)).
```

Comment:
why is this a pointer here?

Reply:
Because it can be nilable I believe. We were printing the pointer instead of the uint64 itself in the logs